src/utils/safe-focus-visible-polyfill.js

```diff
@@ -0,0 +1 @@
+if (process.client) require('focus-visible')
```

Comment:
`focus-visible` breaks SSR so we need to guard it to only be imported on the client.